docs/onboarding/minishift-getting-started.adoc

```diff
@@ -0,0 +1,141 @@
+[[minishft-getting-started]]
+= Getting started with minishift
+
+
```

Comment:
I think some context here would be good. Something like:

```
The aim of this document is to get you up and running with a mobile enabled OpenShift. It will walk you through installing all the base pieces and required configuration. In other guides we will show you how to make use of this installation by representing your mobile clients as resources in OpenShift and provision mobile enabled services, simply and easily using the OpenShift Catalog along with consuming these services from your mobile clients.

```